src/controllers/videoosd.js

```diff
@@ -1254,6 +1254,12 @@ define(["playbackManager", "dom", "inputmanager", "datetime", "itemHelper", "med
 
         if (browser.touch) {
             dom.addEventListener(view, "dblclick", onDoubleClick, {});
+        } else {
+            dom.addEventListener(view, "dblclick", function () {
```

Comment:
This is just a style suggestion that I find easier to read.
```suggestion
            var options = { passive: true };
            dom.addEventListener(view, "dblclick", function () { playbackManager.toggleFullscreen(currentPlayer); }, options);
```

Reply:
I like that suggestion, thanks. Tho, I'm pretty sure this function is pure JS without any jQuery